lang/fr/validation.php

```diff
@@ -0,0 +1,187 @@
+    'multiple_of' => 'L\'attribut :attribute doit être a multiple of :value.',
+    'not_in' => 'L\'attribut sélectionné :attribute est invalide.',
+    'not_regex' => 'Le format de l\'attribut :attribute est invalide.',
+    'numeric' => 'L\'attribut :attribute doit être un nombre.',
+    'password' => [
+        'letters' => 'L\'attribut :attribute doit contenir au moins une lettre.',
+        'mixed' => 'L\'attribut :attribute doit contenir au moins une majuscule et une minuscule.',
+        'numbers' => 'L\'attribut :attribute doit contenir au moins un chiffre.',
+        'symbols' => 'L\'attribut :attribute doit contenir au moins un symbole.',
+        'uncompromised' => 'The given :attribute has appeared in a data leak. Please choose a different :attribute.',
+    ],
+    'present' => 'L\'attribut :attribute doit être présent.',
+    'prohibited' => 'L\'attribut :attribute field is prohibited.',
+    'prohibited_if' => 'L\'attribut :attribute field is prohibited quand :other is :value.',
+    'prohibited_unless' => 'L\'attribut :attribute field is prohibited unless :other is in :values.',
+    'prohibits' => 'L\'attribut :attribute field prohibits :other from being present.',
+    'regex' => 'L\'attribut :attribute format est invalide.',
+    'required' => 'L\'attribut :attribute est requis.',
+    'required_array_keys' => 'L\'attribut :attribute field doit contenir entries for: :values.',
+    'required_if' => 'L\'attribut :attribute est requis quand :other is :value.',
+    'required_if_accepted' => 'L\'attribut :attribute est requis quand :other is accepted.',
+    'required_unless' => 'L\'attribut :attribute est requis unless :other is in :values.',
+    'required_with' => 'L\'attribut :attribute est requis quand :values is present.',
+    'required_with_all' => 'L\'attribut :attribute est requis quand :values are present.',
+    'required_without' => 'L\'attribut :attribute est requis quand :values is not present.',
+    'required_without_all' => 'L\'attribut :attribute est requis quand none of :values are present.',
+    'same' => 'L\'attribut :attribute et :other doivent être égaux.',
+    'size' => [
+        'array' => 'L\'attribut :attribute doit contenir :size éléments.',
+        'file' => 'L\'attribut :attribute doit être :size kilooctets.',
+        'numeric' => 'L\'attribut :attribute doit être :size.',
+        'string' => 'L\'attribut :attribute doit être :size caractères.',
+    ],
+    'starts_with' => 'L\'attribut :attribute must start with one of the following: :values.',
+    'string' => 'L\'attribut :attribute doit être une chaîne de caractères.',
+    'timezone' => 'L\'attribut :attribute doit être un fuseau horaire valide.',
+    'unique' => 'L\'attribut :attribute a déjà été utilisé.',
+    'uploaded' => 'L\'envoi de :attribute a échoué.',
+    'url' => 'L\'attribut :attribute doit être un URL valide.',
+    'uuid' => 'L\'attribut :attribute doit être un UUID valide.',
```

Comment:
You could have used `""` strings instead of backslashes to escape the simple quotes ^^